### PR TITLE
Comply with requested changes in #4524

### PR DESCRIPTION
**Pull request by NoooneyDude**
- Adds the requested changes in PR #4524. Partially fixes part 16. of #4654.
- Small tweaks to conform to project standards.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Objects/Disposals/DisposalBin.cs [hunk 1]
```diff
@@ -35,28 +35,33 @@ public enum BinOverlaySprite
 	[ExecuteInEditMode]
 	public class DisposalBin : DisposalMachine, IServerDespawn, IExaminable, ICheckedInteractable<MouseDrop>
 	{
-		const int CHARGED_PRESSURE = 600; // kPa
-		const int AUTO_FLUSH_DELAY = 2;
-		const float ANIMATION_TIME = 1.3f; // As per sprite sheet JSON file.
+		private readonly int CHARGED_PRESSURE = 600; // kPa
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Objects/Disposals/DisposalBin.cs]: why can't these be configurable on inspector?

Comment (NoooneyDude) [reply UnityProject/Assets/Scripts/Objects/Disposals/DisposalBin.cs]: Hmm, I tend to use constants when the value, once found, shouldn't be changed unless the code changes too. Though, some of these probably could be serialized.

Comment (corp-0) [reply UnityProject/Assets/Scripts/Objects/Disposals/DisposalBin.cs]: in that case, make it ``const`` or was codacy complaining? It should only complain about publics const that are used in other classes.

Comment (NoooneyDude) [reply UnityProject/Assets/Scripts/Objects/Disposals/DisposalBin.cs]: You're right. Will fix.

**Pull request merged**